packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/models/index.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export {
+  Test,
+  PassFailCriteria,
+  PassFailMetric,
+  PFMetrics,
+  PFAgFunc,
+  PFAction,
+  PFResult,
+  Secret,
+  SecretType,
+  CertificateMetadata,
+  CertificateType,
+  LoadTestConfiguration,
+  OptionalLoadTestConfig,
+  TestInputArtifacts,
+  FileInfo,
+  FileType,
+  FileStatus,
+  TestAppComponents,
+  AppComponent,
+  TestServerMetricConfig,
+  ResourceMetric,
+  FileInfoList,
+  TestsList,
+  TestRun,
+  ErrorDetails,
+  TestRunStatistics,
+  TestRunArtifacts,
+  TestRunInputArtifacts,
+  TestRunOutputArtifacts,
+  PFTestResult,
+  Status,
+  MetricDefinition,
+  NameAndDesc,
+  AggregationType,
+  MetricUnit,
+  MetricAvailability,
+  TimeGrain,
+  MetricNamespace,
+  DimensionFilter,
+  TimeSeriesElement,
+  MetricValue,
+  DimensionValue,
+  Interval,
+  DimensionValueList,
+  CreateOrUpdateTestOptions,
+  CreateOrUpdateAppComponentsOptions,
+  CreateOrUpdateServerMetricsConfigOptions,
+  GetAppComponentsOptions,
+  GetServerMetricsConfigOptions,
+  GetTestOptions,
+  GetTestFileOptions,
+  ListTestFilesOptions,
+  ListTestsOptions,
+  UploadTestFileOptions,
+  DeleteTestFileOptions,
+  DeleteTestOptions,
+} from "../loadTestAdministration/models/index.js";
+export {
+  Test as LoadTestRunClientTest,
```

Comment:
This is the shared model between the two sub clients loadTestRun and loadTestAdministration, This is temporarily rename it with a subclient prefix. I am going to address it in #1892